src/application/extension-popup-menu/app.tsx

```diff
@@ -0,0 +1,55 @@
+import { Route, Routes } from 'react-router-dom';
+
+import { useExtensionSettings } from 'shared/extension';
+import { Closable } from 'shared/ui';
+
+import { Footer } from './footer';
+import { PopupHomeView } from './popup-home-view';
+import { TopBar } from './top-bar';
+import { CustomizationSettings } from './settings-view/customization-settings';
+import { MainSettingsMenu } from './settings-view/main-settings-menu';
+
+export const EXTENSION_POPUP_MENU_ROUTES = {
+  HOME: '/',
+  SETTINGS: {
+    MAIN_VIEW: '/settings',
+    CUSTOMIZATION: '/settings/customization',
+  },
+};
```

Comment:
I can imagine situation where we want to open given route by default from outside of popup, in this case we would need to also consume routes, we already discussed it on discord that it might be worth to open wallet connect modal inside popup, maybe we could move that const into `shared/extension` ?